dist/aframe-teleport-controls.js

```diff
@@ -52,12 +52,17 @@
 	  throw new Error('Component attempted to register before AFRAME was available.');
 	}
 
+	var buttons = ['trackpad', 'trigger', 'grip', 'menu', 'thumbstick', 'A', 'B', 'X', 'Y', 'surface', 'point', 'pointing', 'pistol', 'thumb'];
```

Comment:
supporting hand-controls was the point of the fork, as mentioned above, so sounds as though you may not be inclined to take the changes and I may need to maintain the fork...